pkgs/gitleaks/gitleaks/registry.yaml

```diff
@@ -13,7 +13,7 @@ packages:
         format: zip
     replacements:
       amd64: x64
-      arm64: x64
+      aarch64: arm64
```

Comment:
aarch64 is an invalid key.

Reply:
I mean keys of `replacements` must be one of `amd64`, `arm64`, `windows`, `linux`, and `darwin`.